tests/unit/test_cli_runner_aggregation.py

```
@@ -0,0 +1,393 @@
+from aiperf.common.models.export_models import JsonMetricResult
+from aiperf.orchestrator.models import RunResult
+
+
+class TestAggregatePerSweepValue:
+    """Test per-value aggregation for sweep + confidence mode."""
+
+    @pytest.fixture
+    def mock_results_repeated_mode(self) -> list[RunResult]:
+        """Create mock results for repeated mode sweep + confidence.
+
+        Structure: 2 trials × 3 concurrency values = 6 runs
```

Ruff (0.15.1)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)